src/Sarif/Writers/SarifLogger.cs

```diff
@@ -228,12 +227,9 @@ public virtual void Dispose()
                     _run.Invocations[0].EndTimeUtc = DateTime.UtcNow;
                 }
 
-                // Note: we write out the backing rules
-                // to prevent the property accessor from populating
-                // this data with an empty collection.
```

Comment:
>collection [](start = 43, length = 10)

Good lord, I don't even want to _try_ to remember what that comment might once have meant.